src/aap_eda/core/models/management_job.py

```
@@ -0,0 +1,80 @@
+
+from django.db import models
+
+from aap_eda.core.enums import ExecutionStatus, ManagementJobType
+
+from .base import BaseOrgModel, UniqueNamedModel
+
+__all__ = (
+    "ManagementJob",
+    "ManagementJobSchedule",
+    "ManagementJobExecution",
+)
```

Ruff (0.15.10)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)